docs/source/recipes/lora_finetune_single_device.rst

```diff
@@ -19,43 +18,33 @@ in just two steps:
 
 .. code-block:: bash
 
+    # download the model
     tune download meta-llama/Meta-Llama-3.1-8B-Instruct \
     --output-dir /tmp/Meta-Llama-3.1-8B-Instruct \
     --ignore-patterns "original/consolidated.00.pth"
 
+    # run the recipe
     tune run lora_finetune_single_device \
     --config llama3_1/8B_lora_single_device
 
-You can quickly customize this recipe through the :ref:`cli_label`. For example, when fine-tuning with LoRA, you can adjust the layers which LoRA are applied to,
-and the scale of the imapct of LoRA during training:
+You can customize this recipe through the :ref:`cli_label`. For example, when fine-tuning with LoRA, you can adjust the layers which LoRA are applied to:
 
 .. code-block:: bash
 
     tune run lora_finetune_single_device \
     --config llama3_1/8B_lora_single_device \
-    --model.lora_attn_modules=["q_proj", "k_proj", "v_proj"] \
-    --model.apply_lora_to_mlp=True \
-    --model.lora_rank=64 \
-    --model.lora_alpha=128
+    model.lora_attn_modules=“[q_proj,k_proj,v_proj]” \
+    model.apply_lora_to_mlp=True \
+    model.lora_rank=64 \
+    model.lora_alpha=128
 
-This configuration in particular results in a aggressive LoRA policy which
-will tradeoff higher training accuracy with increased memory usage and slower training.
 
 For a deeper understanding of the different levers you can pull when using this recipe,
 see our documentation for the different PEFT training paradigms we support:
 
 * :ref:`glossary_lora`
 * :ref:`glossary_qlora`
 
-Many of our other memory optimization features can be used in this recipe, too:
```

Comment:
Nit would at least keep this first sentence as a lead-in the the "You can learn more about all of our memory optimization features". Otherwise I agree that it's nice to drop the full list here though